src/libaktualizr/primary/uptane_key_test.cc

```diff
@@ -87,7 +87,7 @@ class UptaneKey_Check_Test {
     // Verify that each secondary has valid keys.
     std::map<Uptane::EcuSerial, std::shared_ptr<Uptane::SecondaryInterface> >::iterator it;
     for (it = sota_client->secondaries.begin(); it != sota_client->secondaries.end(); it++) {
-      EXPECT_TRUE(it->second->sconfig.secondary_type == Uptane::SecondaryType::kVirtual);
+      // EXPECT_TRUE(it->second->sconfig.secondary_type == Uptane::SecondaryType::kVirtual);
```

Comment:
same: just remove it?

Reply:
yeah, I'll just remove it since it won't needed once I move the virtual secondaries on the new way of configuration.